README.md

```diff
@@ -67,6 +67,25 @@ turn makes available the following functions on the server only:
     to be used with [Accounts.onLogin](https://docs.meteor.com/#/full/accounts_onlogin)
     (because it's actually used this way...).
 
+If you need to track changes in the `registered_emails` field, declare your own [`onLogin`](http://docs.meteor.com/#/full/accounts_onlogin) hook. It will be called after the one set by `accounts-emails-field` and will allow you to update whatever data depending on `registered_emails`:
+
+```javascript
+Accounts.onLogin(function (info) {
+	// If login was not successful, quit
+	if (! info.user)
+		return;
+
+	// Get the user, including the registered_emails field added by the "meteor-accounts-emails-field"
```

Comment:
could you please change the package name to `splendido:accounts-emails-field`? ;-)